src/FlightMap/MapItems/VehicleMapItem.qml

```diff
@@ -16,16 +16,18 @@ import QGroundControl               1.0
 import QGroundControl.ScreenTools   1.0
 import QGroundControl.Vehicle       1.0
 import QGroundControl.Controls      1.0
+import QGroundControl.ADSBVehicle   1.0
 
 /// Marker for displaying a vehicle location on the map
 MapQuickItem {
-    property var    vehicle                                                         /// Vehicle object, undefined for ADSB vehicle
-    property var    map
-    property double altitude:       Number.NaN                                      ///< NAN to not show
-    property string callsign:       ""                                              ///< Vehicle callsign
-    property double heading:        vehicle ? vehicle.heading.value : Number.NaN    ///< Vehicle heading, NAN for none
-    property real   size:           _adsbVehicle ? _adsbSize : _uavSize             /// Size for icon
-    property bool   alert:          false                                           /// Collision alert
+    property var     vehicle                                                         /// Vehicle object, undefined for ADSB vehicle
+    property var     map
+    property double  altitude:       Number.NaN                                      ///< NAN to not show
+    property string  callsign:       ""                                              ///< Vehicle callsign
+    property double  heading:        vehicle ? vehicle.heading.value : Number.NaN    ///< Vehicle heading, NAN for none
+    property real    size:           _adsbVehicle ? _adsbSize : _uavSize             /// Size for icon
+    property bool    alert:          false                                           /// Collision alert
```

Comment:
Unclear to me what the change was here, seems like only whitespace

Reply:
Yes, it was for alignment changes that is no longer needed. Removed now.